src/components/dialogs/AdvancedFiltersDialog.js

```diff
@@ -51,12 +69,23 @@ const AdvancedFiltersDialog = ({
   };
 
   const fetchFilters = (params) => fetchCustomFilter(params);
-  
+
   const handleOpen = () => {
-    setFilters(appliedFiltersRowStructure);
+    hasCustomFilters() && isAppliedFiltersRowStructureEmpty()?
+       setFilters(jsonFiltersToRowFilters()) : setFilters(appliedFiltersRowStructure);
     setIsOpen(true);
   };
 
+  const isAppliedFiltersRowStructureEmpty = () => {
+    return !(
```

Comment:
I wonder if we can replace this const definition by: 
```
const isAppliedFiltersRowStructureEmpty = () => {
    const [firstFilter = {}] = appliedFiltersRowStructure;
    return !(firstFilter.filter && firstFilter.field && firstFilter.type);
}
```
It could be simpler in my opinion

Reply:
done